selfdrive/car/toyota/carcontroller.py

```diff
@@ -26,7 +26,7 @@ def update(self, enabled, active, CS, frame, actuators, pcm_cancel_cmd, hud_aler
              left_line, right_line, lead, left_lane_depart, right_lane_depart):
 
     # gas and brake
-    if CS.CP.enableGasInterceptor and enabled:
+    if CS.CP.enableGasInterceptor and CS.CP.openpilotLongitudinalControl and enabled:
```

Comment:
We would still send the CAN packets to the interceptor on line 103. Probably best to combine these two blocks.